### PR TITLE
Update explore endpoints in libs to use v1

### PR DESCRIPTION
**Pull request by jowlee**
### Description
Update explore page to use new v1 endpoints

Relies on updated libs version
reference PRs: 
* https://github.com/AudiusProject/audius-protocol/pull/3353
* https://github.com/AudiusProject/audius-protocol/pull/3354

Fixes PLAT-233

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?
Ran locally against linked libs and discovery running against prod with (fixed) endpoint

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/services/audius-backend/Explore.ts [hunk 1]
```diff
@@ -132,6 +138,21 @@ class Explore {
...
+      const encodedUserId = encodeHashId(userId)
+      const tracks: APITrack[] = await libs().discoveryProvider.getMostLovedTracks(
+        encodedUserId,
+        limit,
+        true
+      )
+      return tracks.map(adapter.makeTrack).filter(removeNullable)
+    } catch (e) {
+      console.error(e)
+      return []
```
Comment (dylanjeffers) [inline packages/web/src/services/audius-backend/Explore.ts:152]: just wondering what the error case is here, i guess seems like the pattern in the file?

**Review by dylanjeffers — APPROVED** [review]
niceee

File: packages/web/src/services/audius-backend/Explore.ts (hunk 1 above)
Comment (jowlee) [reply packages/web/src/services/audius-backend/Explore.ts:152]: Retuning `[]` causes the page to show zero tracks in this playlist - I'm not sure what the intended error behavior is.

**Review by rickyrombo — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/jowlee-explore

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/jowlee-explore

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/jowlee-explore

**Pull request merged**